docs/install-and-setup.md

```diff
@@ -2,6 +2,8 @@
 
 > Using Ionic and the Ionic CLI? Check out these [specific instructions](ionic/cli.md) for Ionic and their CLI.
 
+> Following instructions use `ng add` command that automates several steps for you. If you want to set up things manually refer to [these steps](install-and-setup-manual.md) instead.
```

Comment:
Typos:

* "Following instructions" -> "The following instructions"
* "use `ng add` command" -> "use the `ng add` command"